shell/cgo-enabled.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# This is a small script to determine whether a service manifest has
+# explicitly enabled compiling with CGo, and print out the appropriate
+# value for the CGO_ENABLED environment variable. Defaults to disabled.
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+# shellcheck source=./lib/bootstrap.sh
+source "$DIR/lib/bootstrap.sh"
+
+if [[ "$(yq -r ".arguments.enableCGo" <"$(get_service_yaml)")" == "true" ]]; then
```

Comment:
```suggestion
if [[ "$(yq -r ".arguments.enableCGO" <"$(get_service_yaml)")" == "true" ]]; then
```

I think it technically should be `CGO`...

Reply:
It's unclear to me what the canonical capitalization of it is. It's very inconsistent in https://pkg.go.dev/cmd/cgo.

I guess it's either `Cgo` or `cgo`, so the suggestion should be
```suggestion
if [[ "$(yq -r ".arguments.enableCgo" <"$(get_service_yaml)")" == "true" ]]; then
```